cli/tsc/dts/lib.deno.window.d.ts

```diff
@@ -296,3 +296,6 @@ declare var location: Location;
 
 /** @category Web APIs */
 declare var name: string;
+
+/** @category Web APIs */
+declare var ImageData: any;
```

Comment:
Todo: need to type this correctly.

Is it possible to reuse the class type from `ext/web`?

Reply:
nope, but you can go to the typescript repo and copy the typings from there